docs/Makefile

```diff
@@ -14,10 +14,11 @@ help:
 
 .PHONY: help Makefile
 
-# Like make clean but also remove files generated by autosummary
-cleanall:
-	make clean
+# Like make clean but also remove files generated by autosummary and
+# rendered videos.
+cleanall: clean
 	rm source/reference/*
+	rm -rf source/media
```

Comment:
Why not make the same change in `make.bat`?

Reply:
I don't think that the `make.bat` even has a `cleanall` directive, as of right now. I am not a Windows person, if you know how to write an appropriate equivalent, please make a suggestion.

Reply:
I added `cleanall` in the Makefile, and should have changed make.bat as well